src/improvements/doc/FINANCE_TEMPLATE.md

```diff
@@ -0,0 +1,66 @@
+# Finance Template Usage
+
+The Finance Template allows members of the finance team to send tokens and Ether to multiple addresses in a single transaction.
+
+## Usage
```

Comment:
Let's just copy in a sample TOML file and annotate with comments, e.g. something like this: https://github.com/ethereum-optimism/superchain-ops/pull/739#discussion_r1989893296

I think that will be more clear since the full context of the TOML to work with is visible while reading the docs

Reply:
Ah nice I see the example below. Yea let's just make that the primary docs and add everything else here as comments